jamo/jamo.py

```diff
@@ -146,6 +181,34 @@ def is_hangul_char(character):
     return 0xAC00 <= ord(character) <= 0xD7A3
 
 
+def is_jamo_compound(character):
+    """Test if a single character is a compound, i.e., a consonant 
+    cluster, double consonant, or dipthong.
+    """
+    if len(character) != 1:
+        raise TypeError('is_jamo_compound() expected a single character')
+    if is_jamo(character):
+        character_name = unicodedata.name(character)
+        if "-" in character_name:
```

Comment:
All this seems a bit hacky (esp. the `W`). I would have just made a set of all the compounds and done `if chr in COMPOUNDS` or something like that. However, if it passes the tests I'm okay with this for now.